syne_tune/constants.py

```diff
@@ -15,7 +15,8 @@
 benchmarks.
 """
 
-SYNE_TUNE_FOLDER = "syne-tune"
+SYNE_TUNE_ENV_FOLDER = "SYNETUNE_FOLDER"  # environment variable that allows to overides default library folder
```

Comment:
This might be a bit confusing. Suggestion SYNE_TUNE_FOLDER = "SYNE_TUNE_FOLDER_DEFAULT", "SYNE_TUNE_FOLDER_DEFAULT" = "syne-tune"

Reply:
Thanks, I used your suggestion for the first one. For the second one, think it makes sense to keep `_ENV` to indicate that this is an environment variable.